src/styles/home.scss

```diff
@@ -281,6 +281,76 @@
   @include primary-cta;
 }
 
+// ---------------------------------------------------------------------------
+// Launch Sequences
+// ---------------------------------------------------------------------------
+
+
+  .launch__content {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin: 80px 0 40px 0;
+    width: 100%;
```

Comment:
add some padding-top: (other sections have 130px) above the launch sequence

Reply:
fixed :D